screen/settings/ElectrumSettings.tsx

```
@@ -0,0 +1,544 @@
+            inputAccessoryViewID={DoneAndDismissKeyboardInputAccessoryViewID}
+            isLoading={isLoading}
+          />
+          <BlueSpacing20 />
+          <View style={styles.portWrap}>
+            <View style={[styles.inputWrap, stylesHook.inputWrap]}>
+              <TextInput
+                placeholder={loc.formatString(loc.settings.electrum_port, { example: '50001' })}
+                value={sslPort?.toString() === '' || sslPort === undefined ? port?.toString() || '' : sslPort?.toString() || ''}
+                onChangeText={text => {
+                  const parsed = Number(text.trim());
+                  if (isNaN(parsed)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)